.github/workflows/torchserve-nightly-build.yml

```diff
@@ -6,28 +6,38 @@ on:
     - cron:  '15 11 * * *'
 jobs:
   nightly:
-    runs-on: ubuntu-18.04
+    # creates workflows for the 3 OS: ubuntu, macOS & windows
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Comment:
I think `fail-fast: true` makes more sense in this case since flakes are not allowed in this scenario

Reply:
Done